weave

```diff
@@ -853,6 +910,7 @@ case "$COMMAND" in
         ;;
     reset)
         [ $# -eq 0 ] || usage
+        http_call $CONTAINER_NAME $HTTP_PORT POST /tombstone-self >/dev/null 2>&1 || true
```

Comment:
I thought tombstones were gone. In any case they are an implementation detail, so the HTTP API should use some more generic terminology.